ci/lib/install_functions.inc.sh

```diff
@@ -535,30 +550,43 @@ linux_install() {
 msys_install() {
   local packages=(
     tar
-    zlib-devel
-    libbz2-devel
     git
     automake
     autoconf
     libtool
     automake-wrapper
     gnupg2
     make
-    pkg-config
+    pkg-config 
     mingw64/mingw-w64-x86_64-cmake
-    mingw64/mingw-w64-x86_64-gcc
-    mingw64/mingw-w64-x86_64-json-c
     mingw64/mingw-w64-x86_64-python3
   )
 
+  if [ "${CC-gcc}" = "gcc" ]; then
+    packages+=(mingw64/mingw-w64-x86_64-gcc 
+               mingw64/mingw-w64-x86_64-json-c
+    )
+  else
+   packages+=(clang64/mingw-w64-clang-x86_64-clang 
+              clang64/mingw-w64-clang-x86_64-openmp
+              clang64/mingw-w64-clang-x86_64-libc++
+              clang64/mingw-w64-clang-x86_64-libbotan
+              clang64/mingw-w64-clang-x86_64-libssp
+              clang64/mingw-w64-clang-x86_64-json-c
+              clang64/mingw-w64-clang-x86_64-libsystre
+   ) 
+  fi
+
   pacman --noconfirm -S --needed "${packages[@]}"
 
+  if [ "${CC-gcc}" = "gcc" ]; then
```

Comment:
ditto

Reply:
CC="" will force cmake default which would be gcc. If CC is unset it will also force cmake default but it may fail at this line depending on shell settings.